lib/components/hoc/WithScripts/WithScripts.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+/**
+ *HOC
+ * WithScripts
+ *
+ */
+
+import React from 'react';
+import { getDisplayName } from '../../../core/utils/getDisplayName';
+
+const withScripts = (WrappedComponent, injectScript) => {
+  // ...and returns another component...
+  class WithScripts extends React.Component {
+    constructor(props) {
+      super(props);
+      this.getScriptTag = this.getScriptTag.bind(this);
+      this.scriptLoaded = this.scriptLoaded.bind(this);
+
+      this.state = {
```

Comment:
Can we start using hooks in new components and move away from using Class based components. 

We will need to update templates accordingly.

Reply:
Ohk